src/pineko/scaffold.py

```diff
@@ -36,14 +36,21 @@ def check_folders(configs):
     ----------
     configs : dict
         configs dictionary containing all the paths to be checked
+    Returns
+    -------
+    : tuple(bool, list, dict)
```

Comment:
please check the generated sphinx explicitly ... however, I think this should be the usual syntax:
```
success : bool
    True if the configuration is correct
missing_key : list
    ...
```